package.json

```
@@ -13,6 +13,9 @@
     "prepare": "husky install"
   },
   "dependencies": {
+    "@mdx-js/loader": "^2.3.0",
+    "@mdx-js/react": "^2.3.0",
+    "@next/mdx": "^13.1.6",
```

the depedencies versions should be fixes
```suggestion
    "@mdx-js/loader": "2.3.0",
    "@mdx-js/react": "2.3.0",
    "@next/mdx": "13.1.6",
```